hasura/migrations/carnet_de_bord/1672828442980_fix_nb_beneficiary_for_structure/down.sql

```diff
@@ -0,0 +1,19 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- CREATE OR REPLACE FUNCTION public.nb_beneficiary_for_structure(structure_row structure)
+--  RETURNS bigint
+--  LANGUAGE sql
+--  STABLE
+-- AS $function$
+--     SELECT  count(DISTINCT N.id)
+--     FROM notebook N
+--     LEFT JOIN notebook_member NM on NM.notebook_id = N.id
+--     LEFT JOIN account A on A.id = NM.account_id
+--     LEFT JOIN professional P on P.id = A.professional_id
+--     LEFT JOIN beneficiary B on B.id = N.beneficiary_id
+--     LEFT JOIN beneficiary_structure BS on BS.beneficiary_id = B.id
+--     WHERE
+--         ( BS.structure_id = structure_row.id AND BS.status <> 'outdated')
+--     OR
+--         ( NM.active = true AND P.structure_id = structure_row.id AND BS.structure_id <> structure_row.id AND BS.status <> 'outdated')
+-- $function$;
```

Comment:
Pour la migration `down` il faudrait rétablir la version précédente de la fonction non ? On peut peut-être simplement copier le contenu de `hasura/migrations/carnet_de_bord/1672749193531_add_structure_beneficiary_count/up.sql`:

```suggestion
CREATE OR REPLACE FUNCTION public.nb_beneficiary_for_structure(structure_row structure)
 RETURNS bigint
 LANGUAGE sql
 STABLE
AS $function$
    SELECT  count(DISTINCT N.id)
    FROM notebook N
    LEFT JOIN notebook_member NM on NM.notebook_id = N.id
    LEFT JOIN account A on A.id = NM.account_id
    LEFT JOIN professional P on P.id = A.professional_id
    LEFT JOIN beneficiary B on B.id = N.beneficiary_id
    LEFT JOIN beneficiary_structure BS on BS.beneficiary_id = B.id
    WHERE
        ( BS.structure_id = structure_row.id AND BS.status <> 'outdated' AND (P.structure_id IS NULL OR P.structure_id = structure_row.id))
    OR
        ( NM.active = true AND P.structure_id = structure_row.id AND BS.structure_id <> structure_row.id)
$function$;
```